include/boost/geometry/formulas/vertex_longitude.hpp

```diff
@@ -85,6 +89,8 @@ class vertex_longitude_on_spheroid
         // More constants
         CT const f = flattening<CT>(spheroid);
         CT const C1 = CT(1);
+        CT const C4 = CT(4);
```

Comment:
That's a minor remark. In general variable names should be lower-case names. In other places of the library constants such as these are already lowe-case. In this file below `c0`, `c1` and `c3` are used for constants. So I think that they should be named consistently in this file at least, but it'd probably be good idea to use names consistently across the whole library.